src/components/ItemsList/ItemsList.tsx

```diff
@@ -0,0 +1,15 @@
+import { Item } from '../Item/Item';
+
+interface Props {
+  items: string[];
+}
+
+export const ItemList: React.FC<Props> = ({ items }) => {
```

Comment:
The component name `ItemList` does not match the filename `ItemsList.tsx`. It's important to keep the component name consistent with the filename to avoid confusion and maintain the convention of one component per file.